test/test_hyper.py

```diff
@@ -1097,6 +1101,10 @@ def test_read_compressed_frames(self):
 
         assert received == b'this is test data'
 
+    def test_response_version(self):
+        r = HTTP20Response(HTTPHeaderMap([(':status', '200')]), None)
+        assert r.version is HTTPVersion.http20
+
```

Comment:
Can we also get a few tests for the abstract HTTPConnection object passing this through?